robyn/__main__.py

```diff
@@ -43,9 +72,8 @@ def index():
 
 if __name__ == "__main__":
     app.start()
-
             """
-        )
+            )
```

Comment:
do we need this?

Reply:
I think my IDE did that. Outdated by my changing Dockerfile generation.